.husky/pre-push

```diff
@@ -1,4 +1,8 @@
-#!/bin/sh
+#!/usr/bin/bash
 . "$(dirname "$0")/_/husky.sh"
 
+version=$(node -p "require('./package.json').version")
+sed -i "/^\([[:space:]]*version: \).*/s//\1$version/" $PWD/.values/env/settings.yaml
+
 npm run lint
+npm run migrate:.values
```

Comment:
what if I would like to review result migration before pushing it?
Perhaps push could indicate that a user must migrate values first by running ...

Reply:
Can you explain your question?

I see this workflow:

1. user runs migrate workflow before or not
2. push makes sure workflow has ran: if diff it runs, if same not

So there is no effect from step 1

Reply:
Valid concern, but it asks for user confirmation during the script.